src/test/java/org/broadinstitute/hellbender/engine/CRAMReferenceIntegrationTest.java

```diff
@@ -0,0 +1,84 @@
+package org.broadinstitute.hellbender.engine;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.PrintReads;
+import org.broadinstitute.hellbender.utils.read.SamAssertionUtils;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+public final class CRAMReferenceIntegrationTest extends CommandLineProgramTest{
+
+    private static final File TEST_DATA_DIR = new File("src/test/resources/org/broadinstitute/hellbender/tools");
+
+    @Override
+    public String getTestedClassName() {
+        return PrintReads.class.getSimpleName();
+    }
+
+
+    @Test(dataProvider="testingDataNoRef", expectedExceptions = UserException.class)
+    public void testNoRef(String fileIn, String extOut) throws Exception {
+        final File outFile = File.createTempFile(fileIn + ".", extOut);
+        outFile.deleteOnExit();
+        File ORIG_BAM = new File(TEST_DATA_DIR, fileIn);
+        final String[] args = new String[]{
+                "--input" , ORIG_BAM.getAbsolutePath(),
+                "--output", outFile.getAbsolutePath()
+        };
+        runCommandLine(args);
+    }
+
+    @Test(dataProvider="testingDataNoRefMultipleInputs", expectedExceptions = UserException.class)
+    public void testNoRefMulti(String fileIn1, String fileIn2, String extOut) throws Exception {
+        final File outFile = File.createTempFile(fileIn1 + ".", extOut);
+        outFile.deleteOnExit();
+        File ORIG_BAM1 = new File(TEST_DATA_DIR, fileIn1);
+        File ORIG_BAM2 = new File(TEST_DATA_DIR, fileIn2);
```

Comment:
These `ORIG_BAM` variables (here and below) are misnamed, since they actually represent cram or sam files in these tests. Recommend renaming to something like `readInput1`, `readInput2`, etc.